lms/templates/dashboard/_dashboard_xseries_info.html

```diff
@@ -28,6 +28,7 @@
             %>
             <a class="btn ${xseries_btn_class}" href="${program_data['program_marketing_url']}" target="_blank"
                data-program-id="${program_data['program_id']}" >
+                <span class="sr">${program_data['display_name']}</span>
```

Comment:
From the code, it looks like there might need to be an `&nbsp;` in here too, but I'm not sure where to find an example of this link in the sandbox to confirm.